doc/Language/objects.pod6

```diff
@@ -449,14 +449,27 @@ will itemize; C<$.a> will be equivalent to C<self.a.item> or C<$(self.a)>.
 
 =begin code
 class A {
+    has Int @.numbers;
     has $.x = (1, 2, 3);
-    method b() { .say for self.x; .say for $.x }
+
+    method show-diff() { .say for self.x; .say for $.x }
+
+    method twice  { self.times: 2 }
+    method thrice { $.times: 3    }
+
+    method times($val = 1) { @!numbers.map(* * $val).list }
```

Comment:
Sorry, but I don't see the colon anywhere either here or in the previous example. Shouldn't it be something like method b( $a: )?

Reply:
@JJ I think the point is in `$.times: 3` method call which wasn't possible before 2020.01.

Reply:
@JJ From the documentation on [self](https://docs.raku.org/language/objects#self):

> The colon-syntax for method arguments is only supported for method calls using self, not the shortcut.

Given a method `meth(p1, p2, ..., p3)`, you've always been able to call it on `self` using the colon as precedence dropper (e.g., `self.meth: p1, p2, ..., p3)` but not on [`$`](https://docs.raku.org/language/variables#The_._twigil) (e.g., `$.meth: p1, p2, ..., p3`). With 2020.01, the shortcut form is now also supported.

> Sorry, but I don't see the colon anywhere either here or in the previous example.

```
self.times: 2 # self.times(2)
$.times: 3    # $.times(3)
```

>Shouldn't it be something like `method b( $a: )`?

That's the syntax for declaring an [explicit invocant](https://docs.raku.org/language/objects#Class_and_instance_methods).